src/memfile.c

```diff
@@ -597,7 +597,7 @@ mf_sync(memfile_T *mfp, int flags)
 	    // OpenNT is strictly POSIX (Benzinger)
 	    // Tandem/Himalaya NSK-OSS doesn't have sync()
 	    // No sync() on Stratus VOS
-# if defined(__OPENNT) || defined(__TANDEM) || defined(__VOS__)
+# ifndef HAVE_SYNC
```

Comment:
Can you please switch the condition? `#ifdef HAVE_SYNC` ?

Reply:
Can do. I never liked this negation anyway, just did it to keep the diff easy peasy  to read :)

Reply:
Done